mbzirc_ign/launch/multi_uav_lift_demo.launch.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2022 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from launch import LaunchDescription
+from launch.actions import IncludeLaunchDescription
+from launch.actions.execute_process import ExecuteProcess
+from launch.substitutions import ThisLaunchFileDir
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+from launch_ros.actions import Node
+
+# ros2 launch mbzirc_ign spawn.launch.py name:=quadrotor_1 world:=coast model:=mbzirc_quadrotor x:=0 y:=0.5 z:=0 R:=0 P:=0 Y:=0 gripper:=mbzirc_suction_gripper
+def generate_launch_description():
+    return LaunchDescription([
+        IncludeLaunchDescription(
+            PythonLaunchDescriptionSource(
+                [ThisLaunchFileDir(), '/spawn.launch.py']),
+            launch_arguments={
+                'name': 'quadrotor_1',
+                'world': 'coast',
```

Comment:
I think the `world` name is not used in the example but change to empty_platform to be consistent with the world name in `multi_drone_lift.sdf`  world? Same for the `ros2 launch` comment above